frontend/src/metabase/ui/components/utils/Divider/Divider.styled.tsx

```diff
@@ -2,19 +2,22 @@ import type { MantineThemeOverride } from "@mantine/core";
 
 export const getDividerOverrides = (): MantineThemeOverride["components"] => ({
   Divider: {
+    defaultProps: {
```

Comment:
We need to change this - the divider may or may not be different colors (as demonstrated in this PR) and we really need to think about flexibility when we are making these components